rapid_response_xblock/block.py

```diff
@@ -93,8 +94,25 @@ def student_view_aside(self, block, context=None):  # pylint: disable=unused-arg
         fragment.initialize_js("RapidResponseAsideInit")
         return fragment
 
+    @XBlockAside.aside_for('studio_view')
+    def studio_view_aside(self, block, context=None):  # pylint: disable=unused-argument
```

Comment:
Are we not checking `is_staff` here like we are above because that doesn't matter for studio views? Just double checking

Reply:
Correct. I would check for it anyway, but I don't think we can actually pull user information from the runtime that gets attached to blocks in studio